student-test/assignment1/Test_validate_username.c

```diff
@@ -14,9 +14,12 @@
 */
 void test_validate_my_username()
 {
+	const char *act=my_username();
+	const char *exp=malloc_username_from_conf_file();
+	TEST_ASSERT_EQUAL_STRING_MESSAGE(exp,act,"te");
```

Comment:
@VenkatTata you should free the malloc'd string from `malloc_username_from_conf_file` here.